docker-compose-rag-doll-only.yml

```diff
@@ -2,9 +2,16 @@ services:
   assistant:
     build: assistant
     environment:
-      ASSISTANT_ROLE: ${ASSISTANT_ROLE}
-      RAG_PROMPT: ${RAG_PROMPT}
-      RAGLESS_PROMPT: ${RAGLESS_PROMPT}
+      ASSISTANT_LANGUAGES: ${ASSISTANT_LANGUAGES}
+      SYSTEM_PROMPT_en: ${SYSTEM_PROMPT_en}
+      RAG_PROMPT_en: ${RAG_PROMPT_en}
+      RAGLESS_PROMPT_en: ${RAGLESS_PROMPT_sw}
+      SYSTEM_PROMPT_sw: ${SYSTEM_PROMPT_sw}
+      RAG_PROMPT_sw: ${RAG_PROMPT_sw}
+      RAGLESS_PROMPT_sw: ${RAGLESS_PROMPT_sw}
+      SYSTEM_PROMPT_fr: ${SYSTEM_PROMPT_fr}
+      RAG_PROMPT_fr: ${RAG_PROMPT_fr}
+      RAGLESS_PROMPT_: ${RAGLESS_PROMPT_fr}
```

Comment:
`RAGLESS_PROMPT_` seems should be `RAGLESS_PROMPT_fr`?

Reply:
Good catch, thank you @anjarakvo .

Fixed now.